RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h

```diff
@@ -0,0 +1,36 @@
+#ifndef __RecoLocalCalo_HGCalRecAlgos_RecHitTools_h__
+#define __RecoLocalCalo_HGCalRecAlgos_RecHitTools_h__
+
+#include <array>
+#include <cmath>
+
+class HGCalGeometry;
+class HGCalDDDConstants;
+class DetId;
+
+namespace edm {
+  class Event;
+  class EventSetup;
+}
+
+namespace hgcal {
+  class RecHitTools {    
```

Comment:
In the future, this will also likely deal with position measurement corrections and other things similar to what's in RecoEcal/LazyClusterTools, but this isn't in place yet. I would prefer to not move the class around as a function of time.